acceptance/apps/rails_test.go

```diff
@@ -130,6 +130,7 @@ var _ = Describe("RubyOnRails", func() {
 		out, err := env.Epinio(rails.Dir, "apps", "push",
 			"--name", rails.Name,
 			"--bind", serviceName)
+		env.OnStageFailureShowStagingLogs(err, out, rails.Name)
```

Comment:
It's unfortunate that we have to copy it everywhere. But I guess a more generic `FailHandler` (https://github.com/epinio/epinio/blob/tekton-removal/acceptance/suite_test.go#L69-L82) wouldn't have access to the context.

Maybe having a more specialized `env.EpinioPush(...)`  that includes this functionality would work.

Reply:
Will think about this while doing the small(er) stuff first. ... At first thought the `EP` would get the name as separate argument, and add the `--name ...` option internally. Then we have all the information needed properly in separate arguments.

Reply:
See #1177